### PR TITLE
Odin nyt clone

### PR DESCRIPTION
**Pull request by bimbolabuari**
# Description

 > This project is [an Odin assignment](https://www.theodinproject.com/courses/html-and-css/lessons/positioning-and-floating-elements) about cloning the [New York Times article page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0) that demonstrates the knowledge of HTML, embedding images and videos using iframe, CSS Grid properties for layout, CSS flexbox properties used for layout, CSS positioning such as absolute and fixed was used positioning elements, and also CSS float.

## Technologies used 

- HTML
   - Semantic HTML
   - Iframe
   - Img tags

- CSS
   - CSS Grid for effective layout
   - CSS Flexbox for effective layout
   - CSS Float
   - CSS positioning 
   - Background properties
   - Display properties 

- Mark-down
- [Prettier](https://prettier.io/) for code formatting
- [Styelint](https://stylelint.io/) for CSS linting
- [HTML Validator](https://validator.w3.org/) and [Webhint](https://webhint.io/)
- [Github Actions for workflow](https://github.com/features/actions)
- [Netlify for deployment](https://app.netlify.com/) 

Screenshot

## Project live link

[Project link](https://deploy-preview-1--bimbola-nyt-clone.netlify.app/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,1028 @@
...
+        <div class="list-flex-container">
+          <ul>
+            <li class="list-header">News</li>
+            <li class="list"><a class="list" href="#">Home Page</a></li>
+            <li class="list"><a class="list" href="#">World</a></li>
+            <li class="list"><a class="list" href="#">U.S.</a></li>
+            <li class="list"><a class="list" href="#">Politics</a></li>
+            <li class="list"><a class="list" href="#">Election 2020</a></li>
+            <li class="list"><a class="list" href="#">New York</a></li>
+            <li class="list"><a class="list" href="#">Business</a></li>
```
Comment (bolah2009) [inline index.html:850]: Let's work more on the footer. There are a few things that need adjustment.
1. The footer has the new york times logo as a header, also there link aligned to the right with text "Go to Home Page »"
2. Make all the sub-headers upper case, we can use [CSS text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) for this.
3. Add spacing to the right-most links with border.

- See how the original page looks below:



| Original page | Clone page |
| :-----: | :-----: |
|image | image |

File: README.md [hunk 2]
```diff
@@ -1,34 +1,43 @@
...
+- Img tags
+
+CSS
+
+- CSS Flexbox for effective layout
+- CSS Float
+- CSS positioning
+- Background properties
+- Display properties
+- Mark-down
```
Comment (bolah2009) [inline README.md:22]: Did you list "mark-down" under CSS?

File: README.md [hunk 3]
```diff
@@ -1,34 +1,43 @@
...
-- Major languages,
-- frameworks,
-- technologies used
+HTML
+
+- Semantic HTML
+- Iframe
+- Img tags
+
+CSS
```
Comment (bolah2009) [inline README.md:15]: You also used [CSS Grid Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout), why not add it?

File: css/index.css [hunk 4]
```diff
@@ -1,3 +1,601 @@
...
+  grid-template-rows: repeat(4, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 11px;
+  margin-top: 10px;
+}
+
+.background {
+  max-width: 100%;
+  grid-area: 1 / 1 / -1 / -1;
+}
```
Comment (bolah2009) [inline css/index.css:335]: Nice job using CSS grid! 👍🏾 💯

**Review by bolah2009 — APPROVED** [review]
Excellent Job with the project.

Please merge after making the suggested changes. 🚀

File: index.html [hunk 5]
```diff
@@ -3,7 +3,1028 @@
...
+              “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer
+              who was not part of the team, “you wind up at inflation.”
+            </p>
+          </div>
+        </div>
+
+        <div>
+          <div class="version">
+            A version of this article appears in print on
+            <p>March 18, 2014</p>
```
[the remark this answers is not in the document]
Comment (bimbolabuari) [reply index.html:583]: I just observed that, thanks I will work on it Immediately

File: index.html [hunk 6]
```diff
@@ -3,7 +3,1028 @@
...
+                      <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M5.987 2.772l.025.425-.429-.052c-1.562-.2-2.927-.876-4.086-2.011L.93.571.784.987c-.309.927-.111 1.906.533 2.565.343.364.266.416-.327.2-.206-.07-.386-.122-.403-.096-.06.06.146.85.309 1.161.223.434.678.858 1.176 1.11l.42.199-.497.009c-.481 0-.498.008-.447.19.172.564.85 1.162 1.606 1.422l.532.182-.464.277a4.833 4.833 0 0 1-2.3.641c-.387.009-.704.044-.704.07 0 .086 1.047.572 1.657.762 1.828.564 4 .32 5.631-.641 1.159-.685 2.318-2.045 2.859-3.363.292-.702.583-1.984.583-2.6 0-.398.026-.45.507-.927.283-.277.55-.58.6-.667.087-.165.078-.165-.36-.018-.73.26-.832.226-.472-.164.266-.278.584-.78.584-.928 0-.026-.129.018-.275.096a4.79 4.79 0 0 1-.755.294l-.464.148-.42-.286C9.66.467 9.335.293 9.163.24 8.725.12 8.055.137 7.66.276c-1.074.39-1.752 1.395-1.674 2.496z"
+                        fill="#000"
+                      ></path>
+                    </svg>
+                  </a>
+                </li>
+
```
[the remark this answers is not in the document]
Comment (bimbolabuari) [reply index.html:621]: WIll make an adjustment on that

File: index.html [hunk 7]
```diff
@@ -3,7 +3,1028 @@
...
+            , Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big
+            Bang’s Smoking Gun<span class="black"
+              >. <a href="#">Order Reprints</a> | <a href="#">Today’s Paper</a> |
+              <a href="#">Subscribe</a>
+            </span>
+          </div>
+          <div class="row">
+            <div class="start">
+              <button class="button" type="button">Read 615 Comments</button>
+            </div>
```
[the remark this answers is not in the document]
Comment (bimbolabuari) [reply index.html:593]: Sure!

File: index.html (hunk 1 above)
Comment (bimbolabuari) [reply index.html:850]: Okay! working it ASAP

File: README.md (hunk 3 above)
Comment (bimbolabuari) [reply README.md:15]: That was a last minute decision, I will add it up immediately

File: css/index.css (hunk 4 above)
Comment (bimbolabuari) [reply css/index.css:335]: Thanks a lot!

**Pull request merged**